app/app.cpp

```diff
@@ -1,11 +1,46 @@
 #include "app.h"
 #include <format>
+#include <sstream>
+#include <iomanip>
+
+using namespace std::string_literals;
 
 namespace vsite::oop::v1
 {
-/*
-* function implementations
-*/
+	std::string to_hex(unsigned int n) {
+        char s[64];
```

Comment:
Ne, koristite [std::format](https://en.cppreference.com/w/cpp/utility/format/format), kao što smo pokazivali na predavanjima.